packages/enzyme-matchers/src/utils/html.js

```diff
@@ -1,19 +1,26 @@
 import instance from './instance';
 /* eslint-disable no-console */
+let consoleObject;
+try {
+  consoleObject = console;
+} catch (e) {
+  // If no global console object is available, set consoleObject to a dummy object.
```

Comment:
Sure, I can move this to a new file - good call.

I've seen a global console object not be declared on some JS engines, e.g. http://www.howtobuildsoftware.com/index.php/how-do/b3ks/java-javascript-reactjs-nashorn-using-react-with-nashorn-throws-referenceerror-console-is-not-defined. It seems the version of V8 we are using to run JS on Android doesn't have a global console object.